depotcommun/settings/production.py

```diff
@@ -0,0 +1,20 @@
+from .base import *
+
+# Database
+# https://docs.djangoproject.com/en/3.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'depot-commun',
+        'USER': 'postgres',
+        'PASSWORD': 'password',
```

Comment:
We'll have to think about removing passwords and so on for production in the future.

Reply:
Sure. Passwords and secret key are properties that should be specified on the server. I think heroku has means to do this. That's what the heroku client library that we are using is meant for isn't it?